index.js

```diff
@@ -17,7 +17,7 @@ var stringEndsWith = String.prototype.endsWith || function(s) {
 };
 
 /**
- * @param {string|object} url - The URL, or the result from url.parse.
+ * @param {string|import('node:url').Url|URL} url - The URL, or the result from url.parse.
```

Comment:
Please update the comment so that it wraps at 80 characters per line.

Or change `import('node:url').Url` to simply `url.Url`.

Reply:
```suggestion
 * @param {string|url.Url|URL} url - The URL, or the result from url.parse.
```